components/apimgt/org.wso2.carbon.apimgt.rest.api.gateway.v2/src/main/resources/gateway-api.yaml

```diff
@@ -139,13 +129,10 @@ paths:
     # Undeploy the API in the Gateway
     #-----------------------------------------------------
     post:
-      security:
-        - OAuth2Security:
-            - apim:api_publish
-      x-wso2-curl: "curl -k -H \"Authorization: Bearer ae4eae22-3f65-387b-a171-d37eaa366fa8\" -X POST \"https://127.0.0.1:9443/api/am/gateway/v1/deploy-api?apiName=PizzaAPI&version=1.0&tenantDomain=registry.com\""
+      x-wso2-curl: "curl -k -H \"Authorization: Basic YWRtaW46YWRtaW4=\" -X POST \"https://127.0.0.1:9443/api/am/gateway/v2/deploy-api?apiName=PizzaAPI&version=1.0&tenantDomain=registry.com\""
       x-wso2-request: |
-        POST https://127.0.0.1:9443/api/am/gateway/v1/undeploy-api?apiName=PizzaAPI&version=1.0&tenantDomain=registry.com
-        Authorization: Bearer ae4eae22-3f65-387b-a171-d37eaa366fa8
+        POST https://127.0.0.1:9443/api/am/gateway/v2/undeploy-api?apiName=PizzaAPI&version=1.0&tenantDomain=registry.com
+        Authorization: Basic YWRtaW46YWRtaW4=
       x-wso2-response: "HTTP/1.1 200 OK"
```

Comment:
With redocly based documents, request response formats are bit different. Shall we use the same way as https://github.com/wso2/carbon-apimgt/blob/56cf5fc41fbc958b07af7ee470327ef4e9c98d84/components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1/src/main/resources/publisher-api.yaml#L189-L192

Reply:
this will handle in a separate pr